### DISCUSSION TITLE
Do I need to use sanitize the output for emails?

### DISCUSSION THREAD

**Discussion by VulcanTech-io** [markedjs/marked#2830] [Q&A]
I am using this library to send emails. (using node mailer) Emails are heavily sanitized by the vendors already, so is there a reason to sanitize the output still? (other than it just being a best practice)

**Comment by UziTech** [comment]
The sanitize option is deprecated and should not be used. If you are running user generated marked markdown through marked you can use dompurify on the output of marked if you want to prevent xss attacks.

If the emails are generated by markdown created by you than there is no need to sanitize it. (Unless you think you may accidentally try to hack your users. 😜)
